packages/glimmer-runtime/lib/compiled/blocks.ts

```diff
@@ -60,15 +60,15 @@ export class EntryPoint extends TopLevelTemplate {
 }
 
 export class Layout extends TopLevelTemplate {
-  constructor(program: Program, symbolTable: SymbolTable, public named: string[], public yields: string[]) {
+  constructor(program: Program, symbolTable: SymbolTable, public named: string[], public yields: string[], public hasPartials: boolean) {
     super(program, symbolTable);
   }
 
-  hasNamedParameters(): boolean {
+  get hasNamedParameters(): boolean {
```

Comment:
Actually, since these objects are immutable, should we just assign to these properties in the constructor?